Breadcrumbs.py

```diff
@@ -189,39 +156,28 @@ def on_selection_modified_async(self, view):
     settings = sublime.load_settings('Breadcrumbs.sublime-settings')
     if settings.get('breadcrumbs_statusbar', True):
       separator = settings.get('breadcrumbs_separator', u' › ')
-      view.set_status('breadcrumbs', separator.join(make_breadcrumbs(view) + ['']))
+      view.set_status('breadcrumbs', separator.join(make_breadcrumbs(view) + ['']).rstrip(separator))
```

Comment:
If you want to strip the last separator, then maybe it would be simpler to not add `+ ['']` in the first place?